ndcube/tests/test_ndcube.py

```diff
@@ -134,19 +144,35 @@
 cube_rotated = NDCube(
     data_rotated,
     w_rotated,
-    mask=mask_cube,
-    uncertainty=uncertainty,
-    missing_axes=[False, False, False],
+    mask=mask_rotated,
+    uncertainty=uncertainty_rotated,
     extra_coords=[('time', 0, u.Quantity(range(data_rotated.shape[0]), unit=u.pix)),
                   ('hello', 1, u.Quantity(range(data_rotated.shape[1]), unit=u.pix)),
                   ('bye', 2, u.Quantity(range(data_rotated.shape[2]), unit=u.pix))])
 
+
+@pytest.mark.parametrize(
+    "nd_cube",[
+    cube,
+    cube_rotated,
+    cubem,
+    cubet])
+def test_wcs_object(nd_cube):
+
+    # Test if wcs object is a BaseLowLevelWCS, instance of SlicedLowLevelWCS object
+    # and has a high level wcs object associated
+    assert isinstance(nd_cube.wcs, BaseLowLevelWCS)
+    assert isinstance(nd_cube.wcs, SlicedLowLevelWCS)
+    assert isinstance(nd_cube.high_level_wcs, HighLevelWCSWrapper)
+
+
 @pytest.mark.parametrize(
     "test_input,expected,mask,wcs,uncertainty,dimensions,world_axis_physical_types,extra_coords",
-    [(cubem[:, 1],
+    [
+     (cubem[:, 1],
       NDCube,
       mask_cubem[:, 1],
-      _wcs_slicer(wm, [False, False, False], (slice(None, None, None), 1)),
+      SlicedLowLevelWCS(cubem.wcs, (slice(None, None, None), 1, slice(None, None, None))),
```

Comment:
You only really need the one `None` here (and in many other places)

Reply:
Okay, I will replace them.